pkg/pillar/cmd/zedagent/handleconfig.go

```diff
@@ -30,6 +32,13 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+const (
+	// Topic for zedagent netdumps of successful config retrievals.
+	netDumpConfigOKTopic = agentName + "-config-ok"
```

Comment:
Right now there isn't. I use similar approach that we have for pubsub topics - adding agentName as prefix to avoid collisions between microservices. And inside a microservice, one has to be careful to give topic names unique suffixes.

Reply:
Okay, then it might be good idea to create a separate issue for somebody to look into it. What do you think? Because I think it really can get messy with more people collaborating on it and with more microservices emerging

Reply:
It is the same issue as with pubsub where we have much more topics and haven't run into any conflict yet. But I can create something in Jira just to track it and be aware of it.